tidb-cloud/data-service-api-key.md

```
@@ -144,7 +144,7 @@ APIキーの説明またはレート制限を編集するには、以下の手
 
 1.  プロジェクトの[**Data Service**](https://tidbcloud.com/project/data-service)ページに移動します。
 2.  左側のペインで、対象のデータアプリの名前をクリックすると、その詳細が表示されます。
-3.  **API Key**領域で、 **「アクション」**列を探し、削除したいAPIキーの行で**「...」** &gt; **「削除」を**クリックします。
+3.  **API Key**領域で、 **Action**列を探し、削除したいAPIキーの行で**「...」** &gt; **Delete**をクリックします。
```

markdownlint-cli2 (0.23.2)

[warning] Spaces after list markers
Expected: 1; Actual: 2

(MD030, list-marker-space)